README.md

```diff
@@ -1,2 +1,45 @@
-# Recommended Equipment
-Recommends the equipment that you should use for different bosses and activities in the game.
\ No newline at end of file
+# Recommended Equipment OSRS Plugin
+> This plugin enhances your OSRS gameplay by seamlessly integrating
+> Equipment recommendations from the wiki directly into your RuneLite
+> client, and then filtering your bank. 
+> 
+> Streamline your gearing process for bossing and mini-games by
+> leveraging data directly from the OSRS Wiki without leaving RuneLite.
+
+ ## Features  
+ **Recommendations**: The plugin automatically fetches and displays recommended gear setups from the OSRS Wiki.
+**Bank Filtering**: This plugin filters items in your bank to show you only the best gear based on your chosen boss or activity.
+**Easy Navigation**: The intuitive interface allows easy searching and selecting bosses and activities. 
+
+## Plugin Installation  
+### This is an approved 3rd party plugin for Runelite, found in the Plugin Hub
+>Runelite is a popular free, open-source, and super fast client for Old School RuneScape
+>
+>The Plugin Hub is a repository of plugins created and maintained by community members who are not officially affiliated with RuneLite. RuneLite developers verify these plugins to ensure they comply with Jagex's third-party client rules and are not malicious in any other way.
+
+1. **Installing the Plugin**: Open RuneLite and go to the [Plugin Hub](https://github.com/runelite/runelite/wiki/Information-about-the-Plugin-Hub)
+2. **Search for 'Recommended Equipment'**: Find the plugin and click install
+ ## Usage  
+ **Open the Plugin Panel**
+ The new icon ![Recommended Equipment Plugin Icon](https://www.notion.so/image/https://prod-files-secure.s3.us-west-2.amazonaws.com/c0481dec-d52c-4515-b313-ac407fe3aff6/b1d98a01-8632-4fc8-aebc-58ecbaf623b0/RecEquip.png?table=block&id=1b26b467-d09e-40c5-98ae-e7da042eea18&spaceId=c0481dec-d52c-4515-b313-ac407fe3aff6&width=250&userId=a3b6a3fc-2203-49ea-81dc-d4720960d1e4&cache=v2) is on the right side of the RuneLite window, in the list of Runelite plugins. Click it to open the panel. 
```

Comment:
This icon doesn't work for me. Takes me to a login page. You should be able to just reference the one in the repo. Or if you want one bigger, you can drag/drop one into the editor here to upload it.